lib/beaker-puppet/install_utils/windows_utils.rb

```diff
@@ -120,19 +120,24 @@ def install_msi_on(hosts, msi_path, msi_opts = {}, opts = {})
             begin
               # 1641 = ERROR_SUCCESS_REBOOT_INITIATED
               # 3010 = ERROR_SUCCESS_REBOOT_REQUIRED
-              on host, Command.new("\"#{batch_path}\"", [], { :cmdexe => true }), :acceptable_exit_codes => [0, 1641, 3010]
```

Comment:
The quoting should stay here.  If it's in "program files\..." this will fail.

Reply:
@glennsarti The quoting did remain. I changed it to `%{` so that we didn't have backslashes everywhere any longer.

Reply:
Derp Me.    Regex is hard :-)